src/idfConfiguration.ts

```diff
@@ -51,14 +56,67 @@ export function readParameter(param: string) {
   return paramValue;
 }
 
-export async function writeParameter(param: string, newValue) {
+export function chooseConfigurationTarget() {
```

Comment:
Maybe we need to persist the user preference, because its tedious for user to select everytime!